compiler/src/parsing/parser.messages

```diff
@@ -1,525 +1,539 @@
+## as_prefix(id_str) -> AS . id_str [ FROM ]
 ##
 ## The known suffix of the stack is as follows:
-## lbrack ELLIPSIS rbrack
+## AS
 ##
-program: ENUM UIDENT LBRACE LBRACK ELLIPSIS WHILE
+
+Expected an alias.
+
+program: MODULE UIDENT EOL FOREIGN WASM LIDENT COLON UIDENT AS LIDENT YIELD
 ##
-## Ends in an error in state: 958.
+## Ends in an error in state: 752.
 ##
-## data_constructor -> lbrack ELLIPSIS . rbrack lparen option(typs) rparen [ RBRACE EOL COMMA ]
+## foreign_stmt -> FOREIGN WASM id_str COLON typ option(as_prefix(id_str)) . FROM file_path [ SEMI RBRACE EOL EOF ]
 ##
 ## The known suffix of the stack is as follows:
-## lbrack ELLIPSIS
+## FOREIGN WASM id_str COLON typ option(as_prefix(id_str))
 ##
-program: ENUM UIDENT LBRACE LBRACK RBRACK WHILE
+program: MODULE UIDENT EOL FOREIGN WASM LIDENT COLON EOL LIDENT ARROW
 ##
-## Ends in an error in state: 953.
+## Ends in an error in state: 757.
 ##
-## data_constructors -> lbrace lseparated_nonempty_list_inner(comma,data_constructor) . option(comma) rbrace [ SEMI EOL EOF COMMA ]
-## lseparated_nonempty_list_inner(comma,data_constructor) -> lseparated_nonempty_list_inner(comma,data_constructor) . comma data_constructor [ RBRACE EOL COMMA ]
+## foreign_stmt -> FOREIGN WASM id_str COLON eols typ . option(as_prefix(id_str)) FROM file_path [ SEMI RBRACE EOL EOF ]
 ##
 ## The known suffix of the stack is as follows:
-## lbrace lseparated_nonempty_list_inner(comma,data_constructor)
+## FOREIGN WASM id_str COLON eols typ
 ##
-program: ENUM UIDENT LBRACE LBRACK WHILE
+program: MODULE UIDENT EOL FOREIGN WASM LIDENT COLON EOL UIDENT AS LIDENT YIELD
 ##
-## Ends in an error in state: 957.
+## Ends in an error in state: 758.
 ##
-## data_constructor -> lbrack . rbrack [ RBRACE EOL COMMA ]
-## data_constructor -> lbrack . ELLIPSIS rbrack lparen option(typs) rparen [ RBRACE EOL COMMA ]
+## foreign_stmt -> FOREIGN WASM id_str COLON eols typ option(as_prefix(id_str)) . FROM file_path [ SEMI RBRACE EOL EOF ]
 ##
 ## The known suffix of the stack is as follows:
-## lbrack
+## FOREIGN WASM id_str COLON eols typ option(as_prefix(id_str))
 ##
-## WARNING: This example involves spurious reductions.
-## This implies that, although the LR(1) items shown above provide an
-## accurate view of the past (what has been recognized so far), they
-## may provide an INCOMPLETE view of the future (what was expected next).
-## In state 19, spurious reduction of production lbrack -> LBRACK
+program: MODULE UIDENT EOL FOREIGN WASM LIDENT COLON LIDENT ARROW
+##
+## Ends in an error in state: 749.
+##
+## foreign_stmt -> FOREIGN WASM id_str COLON typ . option(as_prefix(id_str)) FROM file_path [ SEMI RBRACE EOL EOF ]
+##
+## The known suffix of the stack is as follows:
+## FOREIGN WASM id_str COLON typ
 ##
 
-Expected a list enum declaration using the syntax `enum List<a> { [], [...](a, List<a>) }`.
+Expected the keyword `from` followed by a file path.
```

Comment:
Why does this say `file path`?

Reply:
Because we changed the syntax during development so many times I messed it up 😩